index.html

```diff
@@ -0,0 +1,1147 @@
+                                </h3>
+                                <p>APT Women aims to foster and educate fledgling global business leaders like Chizu Nakamoto.</p>
+                            </div> 
+                        </div>
+                    </div>
+                </div>
+                <div class="sponsored-insight row">
+                    <div class="need-in-center col-sm-12 col-md-5 col-lg-12">
+                        <div class="border">
+                            <img class="full-feat-img img-fluid" src="https://d.newsweek.com/en/full/1576572/shutterstock-680368252.jpg?w=500&h=333&f=72d533330ba92c6f1f08392706a99bf5" alt="Image">
+                            <a class="image-badge sponsored" href="#">SPONSORED INSIGHT</a>
+                            <div class="need-padding">
+                                <h3>
+                                    <a href="#">Cryptocurrencies: A Safe Investment inTimes of Crisis?</a>
+                                </h3>
+                                <p>As the coronavirus is raising havoc across financial markets, investors are looking for ‘safe havens,’ and some are making a case for cryptocurrencies.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- <div id="sponsored-insight" class="border">
+                    <img class="img-fluid" src="https://d.newsweek.com/en/full/1575753/international-institute-management-development-imd.jpg?w=500&h=333&f=61e06c23b210cebda3629ccd2097e2cb" alt="Image">
+                    <a class="image-badge sponsored" href="#">SPONSORED INSIGHT</a>
+                    <div class="need-padding">
+                        <h3>
+                            <a href="#">How Can a Good Business Education Benefit You?</a>
+                        </h3>
+                        <p>Enrolling at a good Business School might be the best investment you can make in your future, and while costs can be significant, the rewards are incalculable. Find out more.</p>
+                    </div>
+                </div> -->
+                <!-- <div id="sponsored-insight" class="border">
+                    <img class="img-fluid" src="https://d.newsweek.com/en/full/1563375/delphian-school.jpg?w=500&h=333&f=c4fe893bfb5642271e1298b1bea3989b" alt="Image">
+                    <a class="image-badge sponsored" href="#">SPONSORED INSIGHT</a>
+                    <div class="need-padding">
+                        <h3>
+                            <a href="#">What Can A Private Education Do For Your Child?</a>
+                        </h3>
+                        <p>What is the single most consequential choice you make to give your child the best possible advantage in life? If you think it’s education, read on.</p>
+                    </div>
+                </div> -->
+                <!-- <div id="sponsored-insight" class="border">
```

Comment:
[REQUIRED] Please remove all the commented codes. If you want to go back, use git command.